### ISSUE TITLE
Final comments box too tall for narrow screens

### ISSUE THREAD

**Issue by Antonialock** [pombase/canto#1784]
When you click the 'submit to curators' button, you click a button to submit to curators.

you are then taken to a page where you can add a note and click another button to 'finish'

on a wide screen laptop the 'finish' button is not visible unless you scroll down.

I fear some people will press the submit button without 'finishing'.

can we:
1. make submitting/finishing a one-step process
2. move the comment textbox somewhere inside main canto
3. check how many sessions have annotation/s that haven't been touched for 3months+? (these people will either have given up - or worst case scenario they thought they submitted the work and it wasn't actually submitted?

**Comment by ValWood** [comment]
yes just spotted this with @CuzickA  and was about to open a ticket:
Alayne thought she had finished because  the header says "curation finished" and "finish" button was not visible

![make box smaller move finish up](https://user-images.githubusercontent.com/7359272/53339940-9fc7d400-38ff-11e9-91b2-b69d7f307bd9.jpg)

I agree that making submission a single step would be better. The text box does not  need to be this big, but i still think we need one here (this is for comments not specific to an annotation)

**Comment by mah11** [comment]
If submission becomes one step, where do we ask for the final comments? I think it would be easy to overlook the box if it doesn't appear front & center somehow, and I'm not sure where I would put it on the existing summary page.

Would it work for the comment box to be in a popup instead of a whole page? I don't love popups, but one might be a decent solution to this problem

**Comment by mah11** [comment]
oh, and is there any desire to edit the wording? that's what the "long winded" phrase suggests ...

**Comment by ValWood** [comment]
~winded~

**Comment by ValWood** [comment]
A quick soultuin, just make the box smaller and move the button up?

**Comment by CuzickA** [comment]
Yes I think we thought the comments box was still useful but too big and it would be helpful to have the 'finish' button higher up on the screen.

**Comment by jseager7** [comment]
In terms of making the 'Finish' button more obvious, I can make the text box shorter, since that should be an easy fix. It will still be possible for the user to vertically resize the box if they run out of space, so I don't think there's any disadvantage to this change.

- - -

With regards to the page being potentially misleading about the curation session being finished, as far as I can tell, **the curation session changes to 'Review' mode as soon as you click 'Submit to Curators' on the summary page.** It seems the curator only has one chance to add any final comments, which isn't ideal.

So the 'Curation finished' heading is accurate, but having another 'Finish' button on that page makes it seem like the session isn't fully complete. We could solve this by renaming the 'Finish' button to 'Submit comments', but I think it might be better to have the session actually not finish until the curator has clicked 'Finish' on the final page.

If we go with the latter solution, we could change the text on the 'Curation Finished' page to make it more obvious that the session isn't finished:

1. 'Submit to curators' &rarr; 'Finalise session'

2. 'Curation finished' &rarr; 'Final comments on curation'

3. 'Your annotations will now be sent …' &rarr; Once you click Finish, your annotations  will be sent …'

**Comment by mah11** [comment]
tl;dr: what if we swap "Finish" and "Submit"?

> it might be better to have the session actually not finish until the curator has clicked 'Finish' on the final page.

Not a bad idea, although the "finished" status referred to is not actually what will make the user-experience difference; it's fairly transparent to users. I don't oppose it, anyway.

I do have some further suggestion for the text amendments: 

1. instead of

> 'Submit to curators' → 'Finalise session'

how about "Finish curation" or "Curation complete"? If neither of these sounds good, I'll try to think of some other possibilities.

"Finalise" doesn't have a more precise meaning than "finish" in the context of Canto curation (or generally), so I don't think it really helps. As a small bonus, if we don't use it, we also dodge a UK/US spelling difference.


2. The heading on the comment-box page could just be "Final comments"; no particular need to include "on curation".

3. Maybe this page is where the button shouldn't say "finish". Actual curation should be finished when they press the button on the previous page (currently labeled "submit"), and it's only after they push the button on this page (the one currently labeled "finish") that the session moves onto the needs-approval list that professional curators review.

So this is the step that intuitively feels like a "submit" action. If we relabel the button on the previous page, this one could become "Submit to curators" or "Submit session".

**Comment by jseager7** [comment]
I agree with the suggestions for amending the text, so the new text replacements would be as follows:

- Curation summary page:
  - Submit to curators &rarr; Finish curation

- Curation finished page:
  - Finish &rarr; Submit to curators
  - Curation finished &rarr; Final comments

The only issue with this is that it doesn't match what Canto actually does at the moment with regards to the session state: currently, the state is changed to `NEEDS_APPROVAL` once the user clicks the 'Submit to curators' button (on the Curation Summary page); this new text implies that change in state should happen after clicking the current 'Finish' button on the Curation Finished page.

So, Canto would need to be changed to change the session state at a different time &ndash; I think this qualifies as a separate issue though, so for now I'll just fix the problem with the size of the comments text box, and I'll open a new issue for the extra changes required for this new text to make sense.

**Comment by mah11** [comment]
re

> The only issue with this is that it doesn't match what Canto actually does at the moment with regards to the session state: 
> ...
> So, Canto would need to be changed to change the session state at a different time – I think this qualifies as a separate issue though

I agree that it can be a separate issue, since this one is mainly about what the users see and do - and they don't need to be (and I'm sure most aren't) aware that at present "the state is changed to NEEDS_APPROVAL once the user clicks the 'Submit to curators' button (on the Curation Summary page)". Non-admin users generally don't see the annotation_status setting for a session.
